src/chaitanyaMankar/ArrayZero.java

```diff
@@ -0,0 +1,37 @@
+package chaitanyaMankar;
+
+import java.util.Arrays;
+
+public class ArrayZero 
+{
+	int[] output;
+	int[] ReturnArray(int[] arr)
+	{
+		int[] output = arr;
```

Comment:
this means both variable pointing to same memory address, if you update any index value then it will change in original array as well